src/arch/cortex-m0/syscalls.S

```diff
@@ -13,12 +13,25 @@
 .thumb_func
```

Comment:
You shouldn't be modifying this (and these modifications are incorrect, they are just an old version of the file)